### ISSUE TITLE
Cannot read property 'set' of undefined

### ISSUE THREAD

**Issue by cayasso** [solkimicreb/react-easy-state#57]
Great library!
For some strange reason I am getting this error in `react-native` whenever I try to import `react-easy-state` 

![image](https://user-images.githubusercontent.com/42518/47536610-564a7800-d87e-11e8-8ada-4106746aed81.png)

Apparently  the descriptor in `batchMethod` is `undefined` 

```
descriptor.set.call(this, batched)
```

Any ideas why this is causing a problem, I am in `react-native@0.57.3` and I am using `react@16.6.0` and `socket.io-client@2.1.1`
I am stuck in a refactoring of an app for this issue :(

Thanks!

**Comment by cayasso** [comment]
Found the problem, in `react-native` the descriptor in the example bellow is `undefined`, not sure if there is a way to get around without overwriting `WebSocket` entirely, but definitely adding check for when descriptor is `undefined` is needed, also it would be good to have some kind of global variable that will skip the batch functionality all together and leave it up to the developer.

```js
const descriptor = Object.getOwnPropertyDescriptor(global.WebSocket.prototype, 'onclose')
console.log(descriptor)
```

**Comment by stephanlough** [comment]
Just submitted a PR #58 for this. Seems like long term this code will eventually go away https://github.com/solkimicreb/react-easy-state/blob/master/src/scheduler.js#L66-L93, so it's a quick fix that doesn't monkey patch WebSocket callbacks on React Native.

**Comment by solkimicreb** [comment]
Thanks for the catch and the fix! It is merged and released with v6.0.5 https://github.com/solkimicreb/react-easy-state/releases/tag/v6.0.5. If you have any more issues with this feel free to reopen the issue. (I am with limited internet now so I may be slow to respond for the next week)
